src/annotation/AdulthoodValidator.java

```diff
@@ -0,0 +1,29 @@
+package annotation;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.time.Period;
+
+public class AdulthoodValidator {
+    public  String [] validate(Object object) {
```

Comment:
This method should return only one message not a list.

Reply:
And also, make it STATIC and you could remove the object of this class in the TEST class.